java-agent/src/nativeMain/kotlin/com/epam/drill/agent/configuration/AgentArguments.kt

```diff
@@ -96,6 +98,16 @@ data class AgentArguments(
             type = packagePrefixes.toType(),
             value = packagePrefixes,
             description = "Configure package prefixes for scanning and instrumentation",
+        ),
+        AgentArguments::coverageRetentionLimitBytes.name to AgentParameter(
+            type = coverageRetentionLimitBytes.toType(),
+            value = coverageRetentionLimitBytes.toString(),
+            description = "Configure limit in bytes for coverage retention in case, when admin is unreachable",
+        ),
+        AgentArguments::sendCoverageIntervalMs.name to AgentParameter(
+            type = sendCoverageIntervalMs.toType(),
+            value = sendCoverageIntervalMs.toString(),
+            description = "Configure sending coverage interval in milliseconds",
```

Comment:
change to
`description = "Coverage sending interval in milliseconds"`

Reply:
done